Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -512,10 +512,13 @@ final class ChatViewController: SLKTextViewController {
 
     func registerTypingEvent(_ subscription: Subscription) {
         typingIndicatorView?.interval = 0
+        let user = AuthManager.currentUser()
 
         SubscriptionManager.subscribeTypingEvent(subscription) { [weak self] username, flag in
             guard let username = username else { return }
 
+            if username == user?.username { return }
```

Comment:
Could be added to the guard statement as well I guessl, would be a little bit cleaner ;)

Reply:
Yep :)